sdk/src/contexts/eth/index.ts

```diff
@@ -297,7 +297,7 @@ export class EthContext<T extends WormholeContext> extends RelayerAbstract {
       const tokenAddr = await this.mustGetForeignAsset(token, sendingChain);
       await this.approve(sendingChain, bridge.address, tokenAddr, amountBN);
       const v = await bridge.transferTokensWithPayload(
-        destContext.parseAddress(tokenAddr),
+        destContext.parseAddress(tokenAddr), // TODO: shouldn't this be the sending context (possible bug)?
```

Comment:
Yes, that's right actually. This method isn't in use right now, I think last time we talked, we discussed not having a separate send/sendWithPayload.  Should just add a payload parameter back to `send`